template/androidApp/src/main/res/values/strings.xml

```diff
@@ -4,6 +4,8 @@
     <!-- Generic -->
     <string name="api_error">Error from API.</string>
 
+    <!-- Accessibility -->
+
```

Comment:
Empty

Reply:
was on purpose, but I can take it out :)